code/modules/vending/toys.dm

```diff
@@ -41,3 +41,31 @@
 /obj/item/vending_refill/donksoft
 	machine_name = "Donksoft Toy Vendor"
 	icon_state = "refill_donksoft"
+
+/obj/machinery/vending/donksofttoyvendor/hugbox
+	desc = "Ages 8 and up approved vendor that dispenses toys. This one feels noticably less fun..."
```

Comment:
"An ages 8 and up approved vendor that dispenses toys. This one feels noticeably less fun..."